frontend/components/Slider.js

```diff
@@ -4,22 +4,28 @@ import withAutoplay from 'react-awesome-slider/dist/autoplay'
 import 'react-awesome-slider/dist/styles.css'
 import './Slider.css'
 
+const images = [
```

Comment:
I'm having second thoughts about this implementation. 
It might actually be more verbose and difficult to work with lol

Reply:
your refactor of code seems great. I think it is better though to have the map function as a JSX directly instead of being in a `const` , because in that case... you don't have to refer what is `imageTags` especially because it is a very short line of code.

in terms of storing the image links... I can't think of anything better than an array... the good thing about it is that it makes the presentation of the images more consistent in a way where I can just change the `map`... and it will reflect the changes with the rest of the images

Reply:
lol these image names are numbered questionably aha but you can dynamically generate this list rather then hardcoding things.
`_.range(1,11).map(x => <div data-src={`images/landing/${x}.jpg`} key={index} />)` here's a one liner to do the same thing you can even put this inside the html

Reply:
Ayy nice job